pkg/gadgets/run/types/metadata.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/btf"
+	"github.com/hashicorp/go-multierror"
+)
+
+type Alignment string
+
+const (
+	AlignmenNone   Alignment = ""
+	AlignmentLeft  Alignment = "left"
+	AlignmentRight Alignment = "right"
+)
+
+type EllipsisType string
+
+const (
+	EllipsisNone   EllipsisType = ""
+	EllipsisStart  EllipsisType = "start"
+	EllipsisMiddle EllipsisType = "middle"
+	EllipsisEnd    EllipsisType = "end"
+)
+
+// FieldAttributes describes how to format a field.
+// Almost 1:1 mapping with columns.Attributes.
+// TODO: Better to use columns.Attributes directly?
```

Comment:
There are a couple of issues when switching to columns.Attributes there:
- The columns package defines those constants int, we'd need to change everything to string so that's understandable by users
- We need to change the meaning of some fields to have a different behavior, like Visible -> Hidden

@flyth what do you think about it?